test/csuite/wt8659_reconstruct_database_from_logs/main.c

```diff
@@ -67,11 +67,8 @@ static void usage(void) WT_GCC_FUNC_DECL_ATTRIBUTE((noreturn));
 static void
 dump_table(const char *home, const char *table, const char *out_file)
 {
-    char buf[1024];
-
-    testutil_snprintf(buf, sizeof(buf), "%s -R -h %s/%s dump %s > %s/%s", wt_tool_path, test_root,
+    testutil_system("%s -R -h %s/%s dump %s > %s/%s", wt_tool_path, test_root,
```

Comment:
@pmacko86 was your suggestion to wrap this function call with `testutil_check` or turning `testutil_system` itself into a function?

Reply:
The original intention was to turn `testutil_system` into a macro, so that when something fails, what's printed to the terminal is a useful reference to what actually failed, e.g., function `dump_table`, line 70. As `testutil_system` is currently a function and the command fails, we instead get something like abort in `testutil_system`, line 739, which is not very useful, because you may not know which call to the function failed.

At the time the ticket was written, we were using `testutil_system` and `system` directly in close to 50-60 places, so let's say if I was running `timestamp_abort` and I got a failure inside `testutil_system`, it was annoying trying to figure out exactly what failed. But we don't use these functions as much anymore as we used to, so this part of the ticket is now less important. But still a nice to have if it's not hard :)

Reply:
fixed https://github.com/wiredtiger/wiredtiger/pull/9981/commits/c106baeebc4d319b1232193a890f4fca2c3c5691 and onwards